occurrences/schema.py

```diff
@@ -965,17 +989,14 @@ def _get_or_create_contact_person(contact_person_data):
     return person
 
 
-def _get_study_levels(study_level_ids):
-    """
-    Get a list of study level instances by using a list of study_level_ids.
-    NOTE: Creation not is not allowed.
-    """
+def _get_instance_list(ModelClass, instance_pks: List[str]):
+
     result = []
-    for study_level_id in study_level_ids:
+    for instance_pk in instance_pks:
```

Comment:
I think `ModelClass.objects.filter(pk__in=instance_pks)` works too

Reply:
It does work, but the filter does not raise an exception and that was the reason why I chose this approach. Should it raise, what do you think?

Reply:
Ah you so want to raise an exception if any id in the list doesn't exist?

Reply:
In my opinion, if you submit a form with an invalid value, an error should be raised. The method is now used with study levels and languages, which both have a quite static list of ids in a string format. I would say it's better that an error is raised and the save process is rejected, if an user wants to select an item that is not supported. Otherwise he would miss the information that some of his inputs wasn't actually saved. So yes, I would say it's better that we raise an exception if any id in the list doesn't exist.

Reply:
Sure, totally valid reason. I didn't think it through, please ignore my suggestion above